### ISSUE TITLE
question on model generation / random seed.

### ISSUE THREAD

**Issue by johndpope** [ciaua/unagan#14]
I trained all day - and ran generator pointing to saved pytorch file / to spit out 5 files. 
    param_fp = f'checkpoints/save/20211112_074506/model/params.Generator.best_Convergence.torch'

is it a case that the model spits out will be same 5 times? is there a random seed that can be swiched in on each run to make different sounds ( explore latent space)?

UPDATE
woops just found this - need to update the readme to make it clear.

    parser.add_argument(
        '--seed',
        dest='seed',
        default=123,
        help='Random seed. Default: 123'
    )

the logic seems flawed - seems like each new sample will carry same seed?

    # ### Process ###
    torch.manual_seed(seed)
    for ii in range(num_samples):
        print(f'Generate sample {ii}')

maybe something like 
```
from random import randint

    for ii in range(num_samples):
        if seed != 123:
            torch.manual_seed(seed)
        else: 
            x = randint(0,1000000)
            torch.manual_seed(x)
```

doesn't seem to make a difference to the end generated mp3.
UPDATE
maybe there's something wrong with my training.
